src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageControl.xaml.cs

```diff
@@ -48,21 +48,20 @@ private void OnClearEnvironmentVariableGridError(object sender, EventArgs e)
             ClearGridError(dataGridEnvironmentVariables);
         }
 
-        private void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
+        private async void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
         {
             if (DataContext != null && DataContext is DebugPageViewModel)
             {
-                Dispatcher.BeginInvoke(new DispatcherOperationCallback((param) =>
+                await ThreadHelper.JoinableTaskFactory
```

Comment:
Note that `VsTaskRunContext` priorities are not necessarily equivalent to Dispatcher ones:
```csharp
        /// <summary>
        /// Runs the task on UI thread using background priority (i.e. below user input).
        /// Tasks are scheduled even while modal dialogs are open.
        /// Tasks are scheduled to occur when no user input is pending, plus a short delay.
        /// Appropriate for short tasks or slightly longer tasks.
        /// </summary>
        UIThreadBackgroundPriority = __VSTASKRUNCONTEXT.VSTC_UITHREAD_BACKGROUND_PRIORITY,

        /// <summary>
        /// Runs the task on UI thread when Visual Studio is idle.
        /// Tasks are not scheduled till most modal dialogs have been dismissed.
        /// Appropriate for very short running tasks.
        /// </summary>
        UIThreadIdlePriority = __VSTASKRUNCONTEXT.VSTC_UITHREAD_IDLE_PRIORITY,
```
I suspect that `UIThreadIdlePriority` is the more equivalent to `DispatcherPriority.Background`. But I'm not sure about behavior when modal dialogs are active.